### PR TITLE
Process opt0

### PR DESCRIPTION
**Pull request by pizza6inch**
### Description

process 0 completed

### Implementation

split champ data into train and test according to the ratio

### Required Hidden Data

nothing

## Checklist

- [x] Detail implementation description at the top of notebook
- [x] Data input and output paths are correct
- [x] Free of spurious comments
- [x] Free of unnecessary notebook updates (e.g. cell executions count)

### CODE REVIEW

**Comment by Vghxv** [comment]
1. use relative path over absolute path
2. remove unnecessary Beep function

**Comment by Vghxv** [comment]
Part of #39

**Comment by xzh0623** [comment]
outputTrainDataPath = "outputTrainDataPath.csv"
outputTestDataPath = "outputTestDataPath.csv"

output path 可以改為存在 ex: /train75，方便後續做dbeta_threshold file
image

**Comment by Vghxv** [comment]
這是 bug
> train_df, test_df = train_test_split(
    df.iloc[:, 1:].T, train_size=train_ratio, random_state=seed
)

除了輸出 train test 的 normal 和 tumor，要考慮 load 兩個的情況嗎

**Comment by Vghxv** [comment]
: 輸入的csv嗎
 
 對

**Comment by xzh0623** [comment]
> 這是 bug
> 
> > train_df, test_df = train_test_split(
> > df.iloc[:, 1:].T, train_size=train_ratio, random_state=seed
> > )
> 
> 另外，除了輸出 train test 的 normal 和 tumor，要考慮 load 兩個的情況嗎

好

**Comment by Vghxv** [comment]
除了輸出 train test 的 normal 和 tumor，要考慮 load 兩個的情況嗎

**Comment by pizza6inch** [comment]
@xzh0623 @OuChiaYun

**Review by xzh0623 — APPROVED** [review]
